### PR TITLE
Release @google-cloud/talent v0.1.0

### PR DESCRIPTION
**Pull request by JustinBeckwith**
This pull request was generated using releasetool.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+[npm history][1]
+
+[1]: https://www.npmjs.com/package/@google-cloud/talent?activeTab=versions
+
+## v0.1.0
+
+03-12-2019 21:57 PDT
+
+First release!
```
Comment (bcoe) [inline CHANGELOG.md]: my colleague CJ used to like putting `reverts prior commit` for the first commit, just to troll people.

Comment (callmehiphop) [reply CHANGELOG.md]: I might steal that

**Comment by callmehiphop** [comment]
@JustinBeckwith let's hammer this so our 404 doc woes disappear!

**Comment by JustinBeckwith** [comment]
> I like that you call out the fact n the README that this library uses semantic versioning ... could we ask folks to use conventional commits pretty soon too 😄

Absolutely!  Instead of asking in the README though, I want to talk about requiring it in a pre-submit check.  I want something like this:
https://probot.github.io/apps/commitlint/

**Pull request merged**